app/views/katello/api/v2/repository_sets/show.json.rabl

```diff
@@ -5,7 +5,7 @@ object @resource
 glue :content do
   node do |content|
     {
-      id: content.cp_content_id,
+      id: content.cp_content_id.to_i,
```

Comment:
does this work if you then pass the id as an int back to enable/disable?

Reply:
My sandbox is a little busted right now but it looks like it is somehow automatically transformed back into a string:

Request Payload
```
{
	"id": 2705,
	"product_id": 56,
	"basearch": "i386",
	"releasever": "5.10"
}
```

Rails log:
```
21:36:21 rails.1   | 2018-03-02T21:36:21 50c58352 [app] [I] Started PUT "/katello/api/v2/products/56/repository_sets/2705/disable" for 192.168.121.1 at 2018-03-02 21:36:21 +0000
21:36:21 rails.1   | 2018-03-02T21:36:21 50c58352 [app] [I] Processing by Katello::Api::V2::RepositorySetsController#disable as JSON
21:36:21 rails.1   | 2018-03-02T21:36:21 50c58352 [app] [I]   Parameters: {"id"=>"2705", "product_id"=>"56", "basearch"=>"i386", "releasever"=>"5.10", "api_version"=>"v2", "repository_set"=>{"id"=>"2705", "product_id"=>"56", "basearch"=>"i386", "releasever"=>"5.10"}}
```

Reply:
If you can write a test that assert that this works, i'm okay with it

Reply:
I just ended up doing this client-side in order to avoid potential complications (and also writing a test in rails)